Kajabi/sage-lib
# Panel - React Docs Update

## Description

Updates Panel docs for React.


## Testing in `sage-lib`

- Navigate to [Panel](http://localhost:4100/?path=/docs/sage-panel--default-story)
- Check that docs are updated

## Testing in `kajabi-products`

(**LOW**) Updates Panel docs for React. No effect on Kajabi Products.
